tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Constants.java

```diff
@@ -36,7 +36,7 @@
   static final String LH_MARKER = "LH:";
   static final String LF_MARKER = "LF:";
   static final String END_OF_RECORD_MARKER = "end_of_record";
-  static final String TAKEN = "-";
+  static final String NOT_TAKEN = "-";
```

Comment:
Maybe this is better as "NEVER_EXECUTED", since it means the branches in a block were never executed, not just taken.

There's a bit of inconsistency with the usages of the words "taken" and "executed" when compared to the docs here: http://ltp.sourceforge.net/coverage/lcov/geninfo.1.php

```
  BRDA:<line number>,<block number>,<branch number>,<taken>`

       Block  number  and  branch  number are gcc internal IDs for the branch.
       Taken is either '-' if the basic block containing the branch was  never
       executed or a number indicating how often that branch was taken.
```
We seem to have reversed the words "taken" and "execution"...